build/tools/avalanche-bridge-faq.md

```diff
@@ -0,0 +1,139 @@
+
+### How is gas estimated?
+The bridge also leverages Chainlink price feeds to get current gas approximations for the Ethereum network. The gas price used is the higher of the Chainlink FASTGAS value and the Geth gas price approximation. The gas price is padded by a few GWEI to help ensure transactions sent by the bridge get included in a block quickly on Ethereum, which benefits the user experience of the bridge.
+
+### How does the airdrop work?
+Users will be airdropped AVAX token when they transfer more than $75 (subject to change) from Ethereum to Avalanche. The actual amount required is slightly less than the amount displayed on the frontend.  However, the bridge checks the USD equivalent at time of the 35th conf, thus the frontend pads the amount displayed slightly in case of price variance. 
+
+### What if I did not receive my airdrop?
+If you haven’t received your airdrop, please confirm that the transfer amount met the minimum threshold requirements.  The current requirements for .1 AVAX are a transfer of at least $75 USD notional equivalent in assets being transferred.
+
+## Security
+### I have heard of the bridge referred to as trustless.  What does this mean?
+ The Avalanche Bridge is trustless in the sense that no one party is able to access any of the funds held as collateral or mint wrapped assets. All transfers across the bridge must be approved by 3 of 4 independent parties (called Wardens). In this sense, when using the bridge you are not trusting any one party to properly transfer your funds.
+
+### What is the role of the wardens?
+The role of the wardens is four fold:
+1. Storing Secret Shares
+2. Indexing Supported Blockchains
+3. Tracking Processed Transactions
+4. Hosting Public Information
+
+A complete breakdown of a Warden’s role and responsibilities will be provided in the upcoming Avalanche Bridge Tech Design article.
+
+### What is the relationship between Ava Labs and the Wardens?
+The wardens are trusted partners of the Avalanche Foundation.  They have a historical record of technical excellence and have a history of working with the platform.
+
+### Has the code been audited? Where are the audit reports?
+Yes, the code for the bridge, warden and smart contracts have been audited by Halborn separately. Audit reports can be found [here](https://github.com/ava-labs/avalanche-bridge-resources/tree/main/SecurityAudits/v1_1).
+
+##Tokens
+
+### How can I add a token to the bridge?
+[Here](https://www.youtube.com/watch?v=vZqclPuPjMw&list=PLRHl-ulWK4-FPRA7SS1OrCOC8cOc2K8sP&index=3) is the tutorial on how to add a token.
```

Comment:
Do we have a text document? Video might be time-consuming, and not well suited for technical documentation.

Reply:
No we don't have one.

Reply:
Isn't this it: [https://github.com/ava-labs/avalanche-bridge-resources#readme](https://github.com/ava-labs/avalanche-bridge-resources#readme) ?

Reply:
I updated and added a new FAQ to make it clearer.